packages/malloy/src/lang/test/find-table-path-walker.spec.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files
+ * (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ * CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import {TestTranslator, markSource} from './test-translator';
+
+test('Table path can be retrieved', () => {
+  const source = markSource`source: flights is DB.table('my.table.flights')
+  source: flights2 is flights extend { dimension: a is astr }`;
+  const doc = new TestTranslator(source.code);
+  const {pathInfo} = doc.tablePathInfo();
+  expect(pathInfo?.length).toBe(1);
```

Comment:
Can you modify this test to actually check the relevant parts of the returned table info?

I would check:
- Connection ID text
- Table Path text
- Position in document

For the position in the document, you can change the `markSource` call to be 
```
const source = markSource`
  source: flights is ${"DB.table('my.table.flights')"}
  source: flights2 is flights extend { dimension: a is astr }
`;
``` 
then compare `source.locations[0]` to `pathInfo[0].range`